src/analytics.ts

```diff
@@ -61,6 +60,9 @@ export function createAnalyticsInstance(options?: Options) {
     const setUrl = (href: string) => _growthbook?.setUrl(href)
     const getId = () => _rudderstack?.getUserId() || _rudderstack?.getAnonymousId()
 
+    // for QA testing purposes
+    window.getMyId = () => getId()
```

Comment:
```suggestion
    window.getMyId = getId
```
this should do the same.